experiments/ClimaCore/sea_breeze/ocean_rhs.jl

```diff
@@ -27,6 +27,17 @@ function hspace_1D(xlim = (-π, π), npoly = 0, helem = 10)
     return space
 end
 
+# Ocean Simulation - Later to live in Oceananigans
+struct OceanSimulation <: ClimaCoupler.AbstractOceanSimulation
```

Comment:
When we use Oceananigans, this will become instantly more important. The simulation wrapper standardizes what the coupler sees and expects from models. A custom `step!` method for an oceananigans simulation would be required.